docs/node/run-your-node/validator-node.mdx

```diff
@@ -294,16 +290,18 @@ consensus:
         - "{{ seed_node_address }}"
 ```
 
-#### Starting Oasis Node for the First Time
+#### Initializing a Node
```

Comment:
Why is this section needed? The node will initialize itself on first startup anyway (as you also note in a later tip)?